typespec-extension/src/code-model-builder.ts

```diff
@@ -781,7 +785,9 @@ export class CodeModelBuilder {
     }
 
     // group ETag header parameters, if exists
-    this.processEtagHeaderParameters(codeModelOperation, op);
+    if (!this.options["disable-etag-grouping"]) {
```

Comment:
Just check `arm` would be enough for now?

Reply:
For now it is. Srikanta mentioned that in the future we may want new SDKs enabled and existing ones disabled(avoid breaking). A flag would be helpful in this case. 

Agree a positive one sounds better. I'll go with `group-etag-headers`.